docs/index.md

```diff
@@ -23,19 +24,17 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-
-
 This documentation is for Apache Flink version {{ site.version_title }}. These pages were built at: {% build_time %}.
 
 Apache Flink is an open source platform for distributed stream and batch data processing. Flink’s core is a streaming dataflow engine that provides data distribution, communication, and fault tolerance for distributed computations over data streams. Flink builds batch processing on top of the streaming engine, overlaying native iteration support, managed memory, and program optimization.
 
 ## First Steps
 
-- **Concepts**: Start with the basic concepts of Flink's [Dataflow Programming Model](concepts/programming-model.html) and [Distributed Runtime Environment](concepts/runtime.html). This will help you understand other parts of the documentation, including the setup and programming guides. We recommended you read these sections first.
+-   **Concepts**: We recommend you start with the basic concepts of Flink's [Dataflow Programming Model](concepts/programming-model.html) and [Distributed Runtime Environment](concepts/runtime.html). This will help you understand other parts of the documentation, including the setup and programming guides.
 
-- **Quickstarts**: [Run an example program](quickstart/setup_quickstart.html) on your local machine or [study some examples](examples/index.html).
+-   **Quickstarts**: [Run an example program](quickstart/setup_quickstart.html) on your local machine or [study some examples](examples/index.html).
 
-- **Programming Guides**: You can read our guides about [basic API concepts](dev/api_concepts.html) and the [DataStream API](dev/datastream_api.html) or the [DataSet API](dev/batch/index.html) to learn how to write your first Flink programs.
+-   **Programming Guides**: You can read our guides about [basic API concepts](dev/api_concepts.html), the [DataStream API](dev/datastream_api.html) or the [DataSet API](dev/batch/index.html) to learn how to write your first Flink programs.
```

Comment:
Oxford comma?

Reply:
@greghogan No, this was an attempt to make the whole passage flow better as it says…

> xxx and yyy or zzz

So I was attempting to seperate out the two parts of the paragraph, better may be…

> **Programming Guides**: You can read our guides about [basic API concepts](dev/api_concepts.html), the [DataStream API](dev/datastream_api.html), and the [DataSet API](dev/batch/index.html) to learn how to write your first Flink programs.

Then it's an oxford comma, but that's stylistic.